### ISSUE TITLE
Missing PG_STAT_STATEMENTS_LIMIT

### ISSUE THREAD

**Issue by ssiahetiong** [CrunchyData/postgres-operator#2244]
```
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_total pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_mean pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=info msg="Error running query on database \"127.0.0.1:5432\": ccp_pg_stat_statements_top_max pq: column \"pg_stat_statements_limit\" does not exist" source="postgres_exporter.go:1356"
time="2021-01-30T10:24:24Z" level=error msg="queryNamespaceMappings returned 3 errors" source="postgres_exporter.go:1474"
```

A new env variable was introduced in: https://github.com/CrunchyData/pgmonitor/blob/master/exporter/postgres/queries_pg_stat_statements_pg13.yml

https://github.com/CrunchyData/postgres-operator/blob/775fd1e8450fab5289e72a79617cc693f7f72c51/bin/crunchy-postgres-exporter/start.sh#L217
This line is incomplete.

**Comment by jkatz** [comment]
@ssiahetiong Please feel free to contribute what you believe the solution is. I would be happy to review.

**Comment by jkatz** [comment]
@ssiahetiong I have gone ahead and merged this patch and backpatched to 4.6. I am going to evaluate the fix for 4.5 as I believe that line also has the `pg_stat_statements` support, though via a different container.

Thanks for the contribution!
